helm/charts/kratos-selfservice-ui-node/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 appVersion: "v0.1.1-alpha.1"
 description: A Helm chart for ORY Kratos's example ui for Kubernetes
 name: kratos-selfservice-ui-node
-version: 0.7.0
+version: 0.7.1
```

This is updated automagically :)

```suggestion
version: 0.7.0
```